### PR TITLE
Add docstrings for thorium modules

### PR DESCRIPTION
**Pull request by techhat**
### What does this PR do?

Add docstrings.
### Tests written?

No.

### CODE REVIEW

File: salt/thorium/check.py [hunk 1]
```diff
@@ -14,6 +14,21 @@ def gt(name, value):
     '''
     Only succeed if the value in the given register location is greater than
     the given value
+
+    USAGE::
```
Comment (rallytime) [inline salt/thorium/check.py]: Can you switch all of these `USAGE::`s to just `USAGE:`? The `::` makes the `.. code-block:: yaml` formatting not add the correct `yaml` tag on the doc examples and formats the docs inconsistently.

File: salt/thorium/file.py [hunk 2]
```diff
@@ -46,7 +46,18 @@
 
 def save(name, filter=False):
     '''
-    Save the register to <salt cachedir>/thorium/saves/<name>
+    Save the register to <salt cachedir>/thorium/saves/<name>, or to an
+    absolute path.
+
+    USAGE::
+
+    code-block:: yaml
```
Comment (rallytime) [inline salt/thorium/file.py]: Needs a `.. ` for the code-block.

File: salt/thorium/local.py [hunk 3]
```diff
@@ -20,6 +20,29 @@ def cmd(
         **kwargs):
     '''
     Execute a remote execution command
+
+    USAGE::
+
+    code-block:: yaml
```
Comment (rallytime) [inline salt/thorium/local.py]: Needs a .. for the code-block.

File: salt/thorium/reg.py [hunk 4]
```diff
@@ -17,6 +17,15 @@
 def set_(name, add, match):
     '''
     Add a value to the named set
+
+    USAGE::
+
+    code-block:: yaml
```
Comment (rallytime) [inline salt/thorium/reg.py]: Needs a .. for the code-block.

**Comment by rallytime** [comment]
@techhat Just a couple of doc formatting fixes and then we can get this in. This should be against the carbon branch, I think too, so I'll need to back-port it.

**Comment by techhat** [comment]
@rallytime, fixed.

**Comment by rallytime** [comment]
Thanks!

**Pull request merged**